docs/UserGuide.adoc

```diff
@@ -448,13 +509,34 @@ Marks the session with the unique session id of 25 as done.
 
 ==== Filter `filter`
 
-TODO: ADD DETAILS ONCE IMPLEMENTED
+Filters undone sessions fulfilling all provided constraints.
```

Comment:
maybe instead of saying undone, you could say sessions that haven't been done yet. It sounds much nicer.